styles/maxspeed.mapcss

```diff
@@ -64,9 +64,12 @@ way[!maxspeed].tracks
 	kothicjs-ignore-layer: true;
 }
 
-way|z9-["construction:railway"="narrow_gauge"],
-way|z9-["construction:railway"="preserved"],
-way|z10-["construction:railway"="rail"]
+way|z9-["railway"="construction"]["construction:railway"="narrow_gauge"],
+way|z9-["railway"="construction"]["construction:railway"="preserved"],
+way|z10-["railway"="construction"]["construction:railway"="rail"],
+way|z9-["railway"="construction"]["construction"="narrow_gauge"],
+way|z9-["railway"="construction"]["construction"="preserved"],
+way|z10-["railway"="construction"]["construction"="rail"]
```

Comment:
I wonder myself why we render narrow gauge railways earlier (zoom level 9) than other `construction=rail` (z10).